framework/console/controllers/BaseMigrateController.php

```diff
@@ -41,6 +41,28 @@
      * or a file path.
      */
     public $templateFile;
+    /**
+     * @var array the template file for generating migration code automatically.
+     * This can be either a path alias (e.g. "@app/migrations/template.php")
+     * or a file path.
+     */
+    public $templateFileGenerators;
+    /**
+     * @var array the fields
+     */
+    public $fields;
+    /**
+     * @var string the primary key name
+     */
+    public $primaryKey = 'id';
+    /**
+     * @var string
+     */
+    public $createAt = 'create_at';
```

Comment:
I don't think these timestamps are that common that they should be there by default or handled in a special way. They're just fields. In quite some cases they aren't needed. For example, `post_tag` junction table would not have these most probably. Same about any disctionary-like tables such as `tag`.